schema/deploy/warehouse/target/data.sql

```diff
@@ -125,7 +125,7 @@ with target_lineage (identifier, lineage) as (
 
         -- Snomed CT targets used by Ellume, Cepheid, UW Retros
         ('http://snomed.info/id/440930009',        'Adenovirus'),
-        ('http://snomed.info/id/1240581000000104', 'Human_coronavirus.2019'),
```

Comment:
This sqitch change won't actually remove this entry from the table, it only adds new ones. That's actually preferable, because there are existing `presence_absence` records that reference that `target_id` (789), so deleting it would throw an error on the foreign key constraint. So I'd recommend not removing this line until all `presence_absence` records referencing `target_id` 789 have been updated to the new `target_id`, at which point it should be safe to remove this line and delete it manually from the table.

Reply:
Also may be worthwhile to add a verify placeholder on warehouse/target/data (to prevent the sqitch message about it missing), similar to the one for warehouse/site/data: https://github.com/seattleflu/id3c-customizations/blob/master/schema/verify/warehouse/site/data.sql

Reply:
Good point on the targets, I'll add that line back and put a Trello card or something up to remind me to do that after the retros are reprocessed. I'll add that verify script today.

Reply:
We'll also need to clear out PA records with the old target once the new one goes in there. It will create new PA entries, so this should clear old records out that have a new record generated (replace second select with delete): 
```
with new_snomed as (
	select sample_id from warehouse.presence_absence where target_id = 952
) select * from warehouse.presence_absence join new_snomed USING (sample_id) where target_id = 789
```